front/app/components/UI/MentionsTextArea/index.tsx

```diff
@@ -241,7 +241,7 @@ const MentionsTextArea = ({
           displayTransform={mentionDisplayTransform}
           markup={'@[__display__](__id__)'}
           onChange={handleOnChange}
-          onFocus={handleOnFocus}
+          onClick={handleOnClick}
```

Comment:
Now the mentions text area only opens on click, not by default.

Reply:
Sorry, reading from top to bottom. This might work instead. Tried and it was working on my side but it might some manual testing on your side too

```
autoFocus={false}
```